### PR TITLE
code fencing src/en/reference-foo.md

### PR DESCRIPTION
**Pull request by petermatulis**
More code/output fencing on

src/en/reference-*.md

#444 updated

Naked ``` fencing removed for some hyperlinks because many did not have it.

### CODE REVIEW

File: src/en/reference-release-notes.md [hunk 1]
```diff
@@ -113,13 +106,15 @@ This release replaces version 1.22.1.
...
-       client-id: <your-client-id>
+```no-highlight
+my-gce:
+ type: gce
+ project-id: <your-project-id>
+ auth-file: <path-to-json-file-from-google>
+ private-key: <your-private-key>
+ client-email: <your-client-email>
+ client-id: <your-client-id>
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: this is yaml, so probably should be ```yaml

File: src/en/reference-release-notes.md [hunk 2]
```diff
@@ -793,39 +801,45 @@ This release replaces version 1.22.1.
...
+    "Type": "Annotations",
+    "Request": "Set",
+    "Params": {
+         "Annotations": { {
+             "EntityTag": a, "Annotations": pairs1
+             },{
+             "EntityTag": b, "Annotations": pairs2
+         } }
+}
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: this is JSON I think?

File: src/en/reference-release-notes.md [hunk 3]
```diff
@@ -793,39 +801,45 @@ This release replaces version 1.22.1.
...
+    "Type": "Annotations",
+    "Request": "Get",
+    "Params": {
+       "Entities": {
+           {Entity {"Tag": a}},
+           {Entity {"Tag": b},
+       }
+    }
+}
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: ditto

File: src/en/reference-release-notes.md [hunk 4]
```diff
@@ -793,39 +801,45 @@ This release replaces version 1.22.1.
...
+{
+    "Results": {
+        {"EntityTag": a, "Annotations": pairs1, "Error": nil},
+        {"EntityTag": b, "Annotations": pairs2, "Error": nil},
 
-          }
-      }
+    }
+}
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: ditto

File: src/en/reference-release-notes.md [hunk 5]
```diff
@@ -1159,7 +1175,9 @@ This release replaces version 1.22.1.
   existed since 1.18.0, it was not applied consistently to KVM containers.
   KVM containers will now use "daily" when environments.yaml is set to:
 
-      image-stream: daily
+```no-highlight
+image-stream: daily
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: yaml

File: src/en/reference-release-notes.md [hunk 6]
```diff
@@ -1130,15 +1140,19 @@ This release replaces version 1.22.1.
...
 
   The local provider skips apt upgrades by default for faster
   provisioning. If you wish to enable upgrades in your local
   development, set ```enable-os-upgrade``` to
   "true" in your environments.yaml:
 
-      enable-os-upgrade: true
+```no-highlight
+enable-os-upgrade: true
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: yaml

File: src/en/reference-release-notes.md [hunk 7]
```diff
@@ -1130,15 +1140,19 @@ This release replaces version 1.22.1.
   You can configure the options in environments.yaml for fast provisioning
   like so:
 
-      enable-os-upgrade: false
-      enable-os-refresh-update: false
+```no-highlight
+enable-os-upgrade: false
+enable-os-refresh-update: false
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: yaml

File: src/en/reference-release-notes.md [hunk 8]
```diff
@@ -1089,7 +1095,9 @@ This release replaces version 1.22.1.
 
   Juju's harvesting behaviour is set through the environments.yaml file.
 
-      provisioner-harvest-mode: <MODE>
+```no-highlight
+provisioner-harvest-mode: <MODE>
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: yaml

File: src/en/reference-release-notes.md [hunk 9]
```diff
@@ -1199,7 +1223,9 @@ This release replaces version 1.22.1.
   state-server will be on a machine with NUMA support, you can set
   ```set-numa-control-policy``` to true in environments.yaml like this:
 
-      set-numa-control-policy: true
+```no-highlight
+set-numa-control-policy: true
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: yaml

File: src/en/reference-release-notes.md [hunk 10]
```diff
@@ -1170,12 +1188,16 @@ This release replaces version 1.22.1.
...
+agent-stream: devel
+```
 
   You can evaluate the next stable version of Juju before it is the default
   version by selecting the "proposed" stream like this:
 
-      agent-stream: proposed
+```no-highlight
+agent-stream: proposed
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: yaml

File: src/en/reference-release-notes.md [hunk 11]
```diff
@@ -1170,12 +1188,16 @@ This release replaces version 1.22.1.
   the most stable versions of Juju. You can set ```agent-stream``` to
   select "devel" streams now to test the unstable versions of Juju:
 
-      agent-stream: devel
+```no-highlight
+agent-stream: devel
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: yaml

File: src/en/reference-release-notes.md [hunk 12]
```diff
@@ -1210,7 +1236,9 @@ This release replaces version 1.22.1.
   is managing the bridge and bringing networks up and down, set the
   ```disable-network-management``` option in environments.yaml to "true":
 
-      disable-network-management: true
+```no-highlight
+disable-network-management: true
+```
```
Comment (evilnick) [inline src/en/reference-release-notes.md]: yaml

**Comment by evilnick** [comment]
I got bored! try and search through for the yaml-type examples!

**Comment by petermatulis** [comment]
Made the changes and removed your inline comments.

**Comment by evilnick** [comment]
yes, this looks better. One thing i forgot to specifically  mention is you need to maintain the indents (two spaces) for code blocks in foldouts, which are used in the release notes page

e.g.
```
^# juju 1.2x
  blah blah blah blah
  ## Xxxx Yyyyy
  
  blah blah
  ``bash
  foo bar
  blah blah
  ``
```
except, obviously with three backticks

I think if you do a render test of those pages, the foldouts will be broken (they may already be broken in dev docs, but *more* broken
 :violin:

**Comment by evilnick** [comment]
can you try and fix this one up ASAP? - there will shortly be additional release notes and it would be good if we could land this first otherwise it will be a mess to sort out

**Comment by petermatulis** [comment]
I'm trying but I'm having trouble. The commits I made with the online editor do not show up locally when I do a 'git pull' making new local changes impossible to push.

**Comment by evilnick** [comment]
use git fetch 'branch'


On 11 June 2015 at 02:56, Peter Matulis <notifications@github.com> wrote:

> I'm trying but I'm having trouble. The commits I made with the online
> editor do not show up locally when I do a 'git pull' making new local
> changes impossible to push.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/juju/docs/pull/463#issuecomment-110967576>.
>



-- 
Nick Veitch
nick.veitch@canonical.com

**Comment by evilnick** [comment]
that looks better. i will merge before you get into more trouble - if there are rendering issues we can patch them up later

**Pull request merged**